locale/en.json

```diff
@@ -1217,6 +1217,5 @@
   "UNKNOWN_ALGORITHM": "Unknown algorithm",
   "UNKNOWN_ALGORITHM_DESCRIPTION": "Unknown algorithm description",
   "FOR_COLON": " for: ",
-  "OPEN_PARENTHESES": " (",
-  "CLOSED_PARENTHESES": ")"
+  "COMMON_NAME_PLUS_SCIENTIFIC_NAME": "{commonName} ({speciesName}"
```

Comment:
speciesName is missing the closing parenthesis.

Reply:
Done in https://github.com/WildMeOrg/codex-frontend/pull/468/commits/e8d5b6bf611de7ed53f1b1a1371db106118aef02.